modules/bvl_feedback/test/TestPlan.md

```diff
@@ -1,10 +1,17 @@
 # Behavioural Feedback Test Plan
 
-1. Behavioural feedback button (notepad in the toolbar) should show up on the following pages:
- * Candidate Profile
- * Instrument List
- * Any instrument
-[Automation Test]
+1. Behavioural feedback button should show up on the following pages:
+ * Candidate Profile  
+ To find the Candiadate profile, first navigate to 'Access Profile', then select the specific Candidate profile from the list.  
+ ![image](images/candidate_list.PNG)  
+ * Instrument List  
+ To find an instrument, first navigate to a specific Candidate Profile, then click on a specific Visit Label.  
+ ![image](images/visit_label.PNG)  
+ A list of Intruments shall be found in this page as shown below.  
```

Comment:
```suggestion
 A list of Instruments is found in this page as shown below.  
```
typo and changing the tone - @haoweiqiu you'll notice we've basically banned "should" and especially "shall" from all our docs - that's a conscious decision for writing style.